drake/doc/ubuntu.rst

```diff
@@ -30,13 +30,18 @@ Install the prerequisites::
     # Ubuntu 14.04 LTS (Trusty)
     sudo apt-get install cmake-curses-gui
 
+    # These are not required for the basic "make" rule.  We suggest that you
+    # wait to do this step until you actually need these tools.  (We intend
```

Comment:
Yup, I agree with that, but I want to thoroughly that before PRing such an edit, so I'll leave it aside for now.